text/0000-impl-only-glob-use.md

```diff
@@ -0,0 +1,59 @@
+- Feature Name: impl-only-glob-use
+- Start Date: 2019-10-12
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Allow `use path::trait * as _` as a way to import implementations of all traits from `path`.
+
+# Motivation
+[motivation]: #motivation
+
+There are a number of crates that export `prelude` modules, meant to be used via `use cratename::prelude::*`.
+Sometimes, these glob imports are only used to bring trait methods into scope.
+Some well-known crates with which this pattern is often used are `rayon` and `gtk`.
+(see [rayon::prelude], [gtk::prelude])
+
+[rayon::prelude]: https://docs.rs/rayon/1.2.0/rayon/prelude/index.html
+[gtk::prelude]: https://docs.rs/gtk/0.7.1/gtk/prelude/index.html
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Like `use module::Tr as _` can be used to import an individual trait's implementations without bringing it into scope, `use module::trait * as _` can be used to import trait implementations for all public traits from `module` into scope.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+`use module::trait * as _` is equivalent to one `use module::Tr as _` statement for each trait `Tr` that `module` publicly defines or re-exports.
```

Comment:
can you add examples how this would interact with import groupings?

Reply:
Sure, done.